### PR TITLE
Revise descriptions, add tags, from aws-greengrass-iot to cabextract.

### PR DESCRIPTION
**Pull request by mvincerx**
Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

**Comment by bryteise** [comment]
merged.

File: bundles/bcache-tools [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # [TITLE]: bcache-tools
-# [DESCRIPTION]: bcache is a way to pair a fast SSD with a slow HDD to create a hybrid disk
+# [DESCRIPTION]: bcache is a way to pair a fast SSD with a slow HDD to create a hybrid disk.
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Tools and Utilities
```
Comment (fenrus75) [inline bundles/bcache-tools]: TAGS are keywords, not sentences, so the word "AND" is not appropriate

it's also the categories that these show up in the UI so lets make sure that these are the ones we want... like how are "Tools" and "Utilities" different things/tabs ?

Comment (bryteise) [reply bundles/bcache-tools]: It was my understanding that the categories in the UI would match, so the category is "Tools and Utilities" in this case. I may have missed something though.

Comment (mvincerx) [reply bundles/bcache-tools]: "Tools and Utilities" is a category per Endre V. 
Chris N., @pixelgeek, may also know.  Not sure why "and" was chosen.

**Comment by fenrus75** [comment]
but they all need to be 1 word

ToolsAndUtilities is fine

**Comment by mvincerx** [comment]
Thank you for the clarification. I will make a cut-over to new usage in current branch (not yet pushed).   
@bryteise. I  plan to circle back on my PRs previous to `bundle-patch-0005` to correct previous usage.
I assume we must also change  to `MultimediaAndGraphics`  from "Multimedia and Graphics".

**Comment by pixelgeek** [comment]
I believe the choice was arbitrary, but Endre had the pen.

Rgds,
Chris


-------- Original message --------

**Comment by mvincerx** [comment]
Please confirm. You're requesting that all multi-word categories in TAGS to be revised as follows:

DeveloperTools
ToolsAndUtilities
MultimediaAndGraphics
DataScience
ProgrammingLanguages
-------------
Other
Productivity
Education
Games 
Security

**Comment by mvincerx** [comment]
Per correspondence with @bryteise, categories are written with spaces between words and appear in each field separated by commands,  as CSVs. 
Example: 
`[TAGS]: Developer Tools, [keyword], [keyword]`
This same practice is reflected in GH-115, GH-114.

**Pull request closed without merging**